xbmc/cores/AudioEngine/Sinks/windows/AESinkFactoryWin.h

```diff
@@ -183,6 +183,7 @@ struct RendererDetail
   unsigned int nChannels;
   unsigned int uiChannelMask;
   bool bDefault;
+  unsigned long m_samplesPerSec; //!< Sample rate of the system default format
```

Comment:
Maybe keep `bool bDefault;` last member of struct (move one line up)

Reply:
sure.